packages/openneuro-components/src/warn-button/WarnButton.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useRef, useEffect } from 'react'
+import './warn-button.scss'
+import { Tooltip } from '../tooltip/Tooltip'
+import { Button } from '../button/Button'
+
+export interface WarnButtonProps {
+  message?: string
+  icon?: string
+  disabled?: boolean
+  tooltip?: string
+  onClick?: () => void
+  lock?: boolean
+  displayOptions: boolean
+  setDisplayOptions: (boolean) => void
+}
+export const WarnButton = ({
+  message,
+  icon,
+  disabled,
+  tooltip,
+  onClick,
+  displayOptions,
+  setDisplayOptions,
+}: WarnButtonProps) => {
+  const viewAction = (
+    <div className="warn-btn-group " role="group">
+      <div className="slide-in">
+        <Button
+          className="btn-warn-component cancel"
+          iconSize="12px"
+          icon="fa fa-times"
+          color="#fff"
+          backgroundColor="#c00342"
+          onClick={() => setDisplayOptions(!displayOptions)}
+          size="xsmall"
+        />
+
+        <Button
+          className="btn-warn-component success"
+          iconSize="12px"
+          icon="fa fa-check"
+          color="#fff"
+          backgroundColor="#00b489"
+          onClick={onClick}
```

Comment:
this should also toggle displayOptions when clicked

Reply:
This isn't in the current WarnButton, but i think it'd be good to add a loading state at some point.

Reply:
created a confirmClick function, let me know if this is a good approach